sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/ServiceResourceDetails.cs

```diff
@@ -15,14 +15,20 @@ internal partial class ServiceResourceDetails
     {
         /// <summary> Initializes a new instance of ServiceResourceDetails. </summary>
         /// <param name="customDocumentModels"> Details regarding custom document models. </param>
-        /// <param name="customNeuralDocumentModelBuilds"> Quota used, limit, and next reset date/time. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="customDocumentModels"/> or <paramref name="customNeuralDocumentModelBuilds"/> is null. </exception>
-        internal ServiceResourceDetails(CustomDocumentModelsDetails customDocumentModels, QuotaDetails customNeuralDocumentModelBuilds)
+        /// <exception cref="ArgumentNullException"> <paramref name="customDocumentModels"/> is null. </exception>
+        internal ServiceResourceDetails(CustomDocumentModelsDetails customDocumentModels)
         {
             Argument.AssertNotNull(customDocumentModels, nameof(customDocumentModels));
-            Argument.AssertNotNull(customNeuralDocumentModelBuilds, nameof(customNeuralDocumentModelBuilds));
```

Comment:
`ServiceResourceDetails.CustomNeuralDocumentModelBuilds`, a new output-only property introduced in service version 2023-02-28-preview, is marked as required in the swagger. For this reason CodeGen makes a null check when deserializing this property, which fails for older service versions. We're fixing it by adding a swagger transform to make it optional.